src/renderers/dom/shared/inputValueTracking.js

```diff
@@ -109,70 +108,61 @@ function trackValueOnNode(node: any, inst: any): ?ValueTracker {
 
 var inputValueTracking = {
   // exposed for testing
-  _getTrackerFromNode(node: ElementWithWrapperState) {
-    return getTracker(ReactDOMComponentTree.getInstanceFromNode(node));
+  _getTrackerFromNode(inst: ElementWithValueTracker) {
+    return getTracker(inst);
   },
 
-  trackNode(node: ElementWithWrapperState) {
+  _getTrackerFromInstance(inst: ReactInstance | Fiber) {
+    return getTracker(ReactDOMComponentTree.getNodeFromInstance(inst));
```

Comment:
Can we not expose it? Since only tests use it and they already have access to DOM node.